steps/hive-setup.sh

```diff
@@ -23,11 +23,12 @@ log_wrapper_message "Listing the tables in Hbase and exporting it to a file curr
 
 sleep 10
 
-aws s3 cp "${python_logger}" .
+aws s3 cp "${python_logger}" /opt/emr/.
+aws s3 cp "${generate_analytical_dataset}" /opt/emr/.
```

Comment:
As we kind of doing some general things here , may its  a better idea to rename file to something generic like adg-setup?